GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -113,6 +124,19 @@ protected override void Execute(GVFSEnlistment enlistment)
                             this.ServiceName,
                             copySubFolders: true);
 
+                        // upgrader
+                        this.CopyAllFiles(
+                            ProductUpgrader.GetUpgradesDirectoryPath(),
+                            archiveFolderPath,
+                            ProductUpgrader.LogDirectory,
+                            copySubFolders: true,
+                            targetFolderName: ProductUpgrader.UpgradeDirectoryName);
+                        this.LogDirectoryEnumeration(
+                            ProductUpgrader.GetUpgradesDirectoryPath(), 
+                            Path.Combine(archiveFolderPath, ProductUpgrader.UpgradeDirectoryName),
+                            ProductUpgrader.DownloadDirectory, 
+                            "downloaded-assets.txt");

```

Comment:
@alameenshah where is this ending up in the zip file? Top level, or inside the GVFS.Upgrade folder? Let's make sure it's inside that folder (not blocking here)

Reply:
It goes inside. 

Screenshots of the folder structure - 

Top Level
<img width="588" alt="screen shot 2018-10-15 at 2 14 03 pm" src="https://user-images.githubusercontent.com/378580/46969633-beca7580-d084-11e8-8fad-0011f73e942a.png">

Inside the GVFS.Upgrader folder
<img width="592" alt="screen shot 2018-10-15 at 2 14 19 pm" src="https://user-images.githubusercontent.com/378580/46969654-ce49be80-d084-11e8-9b5a-9e25dc961357.png">